### ISSUE TITLE
LR Defaults to 1e-6

### ISSUE THREAD

**Issue by nanafy** [victorchall/EveryDream2trainer#102]
I am not sure if I missed a ReadMe or push note, but the latest repo as of today keeps defaulting the LR to 1e-6. I went back to c8c658d181a2999a179f3a5b3321c4c203bb09fe and it works perfectly. Not sure if this is a glitch on my end and I need a fresh install. Thought I'd mention it, incase it ends up being real.

Edit: I found the note about setting the LR in optimizer.json etc. I was setting LR directly from the CLI and my train.json was also not 1e-6; therefore my LR was not being considered.

**Comment by destlaver** [comment]
dude we had the same issue in the same moment, thank you for saving me time lol

**Comment by destlaver** [comment]
also sometimes I get this error and I don't know why: 
UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

**Comment by nanafy** [comment]
Lol I had trained a whole dataset overnight and was super excited to test it out. The results were terrible and now I know why. Glad I could save you some time.

**Comment by victorchall** [comment]
I fixed the defaulting behavior so the main arg "lr" will override the one in optimizer.json which was intended for better compatibility with old configs.

If you want to use optimizer.json LR just delete the LR from your main config file and/or leave it off your CLI arg list.

**Comment by victorchall** [comment]
> also sometimes I get this error and I don't know why: UserWarning: None of the inputs have requires_grad=True. Gradients will be None warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

This happens during the first sample, its just a warning because the model is temporarily flipped to nograd.  It isn't anything to worry about.
